### ISSUE TITLE
bug(cdk/a11y , material/core): Enabling verbatimModuleSyntax causes error TS2748

### ISSUE THREAD

**Issue by parapada73** [angular/components#27036]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

I have enabled `verbatimModuleSyntax` introduced in typescript 5.0.

> Added `"verbatimModuleSyntax": true` to tsconfig.json 

As a result, an error occurred.

`TS2748: Cannot access ambient const enums when 'verbatimModuleSyntax' is enabled`

### Reproduction

Steps to reproduce:

1. Update to Angular 16.
2. Change to typescript 5.0.4.
3. Add `"verbatimModuleSyntax": true` to tsconfig.json .
4. Build

### Expected Behavior

build success !!

### Actual Behavior

Error: node_modules/@angular/cdk/a11y/index.d.ts:729:9 - error TS2748: Cannot access ambient const enums when 'verbatimModuleSyntax' is enabled.     

729         FocusMonitorDetectionMode,
            ~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/@angular/material/core/index.d.ts:430:9 - error TS2748: Cannot access ambient const enums when 'verbatimModuleSyntax' is enabled.

430         RippleState,
            ~~~~~~~~~~~

### Environment

- Angular: **16.0.0**
- CDK/Material: **16.0.0**
- Browser(s): **Chrome 113**
- Operating System (e.g. Windows, macOS, Ubuntu): **win32 x64**

**Comment by parapada73** [comment]
I got a clear answer from @alan-agius4  about using `verbatimModuleSyntax`.
So I will close this issue.

[https://github.com/angular/angular/issues/50154#issuecomment-1540002109](https://github.com/angular/angular/issues/50154#issuecomment-1540002109)
